prime-router/metadata/valuesets/common.valuesets

```diff
@@ -223,4 +231,14 @@
     - code: WR
       display: Weakly reactive
     - code: W
-      display: Worse--use when direction not relevant
\ No newline at end of file
+      display: Worse--use when direction not relevant
+
+- name: hl70103
+  system: HL7
+  values:
+    - code: D
```

Comment:
Amusing that this isn't called "Testing", but I suspect its cause the T is already used for 'Training'